README.md

```diff
@@ -29,28 +35,144 @@ $ git clone git@github.com:openfun/xblock-configurable-lti-consumer.git
 ```
 
 Once the project has been cloned on your machine, you will need to build a
-custom edx-platform docker image that includes the xblock and setup a
-development environment that includes all required services up and running (more
-on this later):
+custom [edx-platform docker image](https://github.com/openfun/openedx-docker) that
+includes the configurable LTI consumer XBlock and setup a development environment that
+includes all required services up and running (more on this later):
 
 ```bash
 $ cd xblock-configurable-lti-consumer
 $ make bootstrap
 ```
 
-If everything went well, you should now be able to access to the following
-services:
+If everything went well, you should now be able to access the following services:
 
 - Open edX LMS: http://localhost:8072
 - Open edX CMS: http://localhost:8082
 
 with the following credentials:
 
 ```
-email: admin@foex.edu
-password: openedx-rox
+email: admin@example.com
+password: admin
+```
+
+## Configuration examples
+
+The following LTI configuration does several things:
+
+- it adds a "Marsha Video" link behind the `Advanced` component button in the Studio to add a
+  video in one click. The video is automatically added to
+  [Marsha](https://github.com/openfun/marsha),
+- it forces all LTI consumer XBlocks that are pointing to `ltiapps.net` to use a modal, together
+  with a few other settings,
+- it forces all other LTI consumer XBlocks to open in a new window and ask before sending the
+  user's username and email.
+
+```yaml
+LTI_XBLOCK_CONFIGURATIONS:
+  # A configuration that adds a Marsha Video XBlock
+  - boilerplate: marsha
+    display_name: Marsha Video
+    oauth_consumer_key: InsecureOauthConsumerKey
+    shared_secret: InsecureSharedSecret
+    fields:
+      lti_id: marsha
+      description: ""
+      launch_target: iframe
+      launch_url: https://marsha.education/lti-video/
+      custom_parameters: []
+      button_text: button
+      inline_height: 100
+      modal_height: 400
+      modal_width: 80
+      has_score: False
+      weight: 0
+      hide_launch: False
+      accept_grades_past_due: False
+      ask_to_send_username: True
+      ask_to_send_email: True
+  # A configuration that applies to all LTI consumer XBlocks pointing to a launch url on the
+  # ltiapps.net domain. It does not add a link to the `Advanced` Component button in the Studio
+  - pattern: .*ltiapps\.net.*
+    fields:
+      launch_target: modal
+      custom_parameters: []
+      button_text: LTI demo
+      inline_height: 100
+      has_score: False
+      weight: 0
+      hide_launch: False
+      accept_grades_past_due: False
+      ask_to_send_username: True
+      ask_to_send_email: True
+  # A configuration that enforces a default behavior for all LTI consumer XBlocks
+  - pattern: .*
+    fields:
+      launch_target: in_window
+      ask_to_send_username: True
+      ask_to_send_email: True
+
```

Comment:
Extra empty row.

Reply:
I replaced the whole thing by a Python version as suggested by @rmoch